### DISCUSSION TITLE
show best play for in game global leaderboard

### DISCUSSION THREAD

**Discussion by N0Y0U2114** [ppy/osu#30945] [Ideas] [answered]
unless your previous score is on the top 50, there is no way to see if you've beaten your previous highest play until results (that im aware of). this would add shpwing your top play to the boTom that would get passed like in local leaderboard mode

**Comment by peppy** [comment, marked as answer]
Should be solved as part of https://github.com/ppy/osu/issues/26331.

**Reply by N0Y0U2114** [reply]
Im referring to the in play leaderboard (Unsure of what its actually called). This is a mock up of said behavior
![image](https://github.com/user-attachments/assets/2b63d402-189c-4280-afe6-aa972349f638)
Showing the highest local score by the player at the bottom of the global leaderboard to be able to tell at a glace how well you are doing compared to said play.

**Reply by peppy** [reply]
yes it's all linked
